bumble/sdp.py

```diff
@@ -872,8 +966,18 @@ async def search_attributes(
     async def get_attributes(
         self,
         service_record_handle: int,
-        attribute_ids: List[Union[int, Tuple[int, int]]],
-    ) -> List[ServiceAttribute]:
+        attribute_ids: list[Union[int, tuple[int, int]]],
```

Comment:
```suggestion
        attribute_ids: list[int | tuple[int, int]],
```

Reply:
Unfortunately, the | syntax for unions is only supported for Python >= 3.10, but we still support Python 3.9